client/src/pages/Paste.js

```diff
@@ -41,6 +42,13 @@ function Paste({ match }) {
           <Author name={paste.author} />
           <PastaPoints pastaPoints={paste.pastaPoints} pasteId={pasteId} />
           <PasteArea>{paste.value}</PasteArea>
+          <EmbedButton
+            hidden={
```

Comment:
`hidden` is not a PropType of `EmbedButton`.

Instead of `hidden`, I would not render the `EmbedButton` at all.